### DISCUSSION TITLE
Hindi (Latin) Capabilities

### DISCUSSION THREAD

**Discussion by kaulsh** [deepgram/community#934] [Feature Requests]
I'll start this off with saying I absolutely love what you guys are doing. The fact that you had Hindi (Latin Script) as a language option ended up being a huge benefit for my product.

I'm curious about what you guys are thinking in terms of prioritization of:
1. Code-switching of english and hindi (latin). Although hi-latin is able to do an acceptable job on the english part, i've noticed when using `en` directly, I get better results on certain statements.
2. Filler words detection for hindi.

**Comment by deepgram-community** [comment]
are you asking how we prioritise the processing of audio, or how we're prioritising future improvements to our model?
> luke

**Reply by kaulsh** [reply]
Prioritizing future improvements. I just want to know if hindi and english code switching and filler words is in your immediate roadmap or whether it's not a priority right now. It would give me some clarity on how I should proceed with things on my end.

**Comment by deepgram-community** [comment]
I've had to defer to our product team. I'll let you know as soon as I have an answer. Do you have a project ID so I can link this request to an account?
> luke

**Reply by kaulsh** [reply]
Thanks. Here you go: 81efb85f-cd88-44a4-8b23-44de6f73ced0
